code/places/.gitignore

```diff
@@ -1 +0,0 @@
-/build
```

Comment:
Don't we need any `.gitignore` for this extension? Or do you plan to add it later?

Reply:
The ignored files for the whole project are listed here 
https://github.com/adobe/aepsdk-places-android/blob/dev-v3.0.0/.gitignore

This is an internal gitignore which only aims to exclude the build folder of the extension